### PR TITLE
OSI-SAF Prep Fix

### PR DESCRIPTION
**Pull request by ShannonShields-NOAA**
## Pull Request Testing ##

- [ ] Describe testing already performed for this Pull Request:</br>
The prep job has been run on the command line and OSI-SAF output has been confirmed to be correct.
- [ ] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions:</br>
Please run the prep ecf script (EVS/ecf/subseasonal/prep/jevs_subseasonal_prep.ecf) to test the prep job.
- [ ] Has the code been checked to ensure that no errors occur during the execution? **[Yes or No]**
Yes (no errors for the prep step)
- [ ] Do these updates/additions include sufficient testing updates? **[Yes or No]**
Yes
- [ ] Please complete this pull request review by **[Fill in date]**.</br>
ASAP (The corrected OSI-SAF files need to start accumulating for the stats to run.)
## Pull Request Checklist ##

- [ ] Review the source issue metadata (required labels, projects, and milestone).
- [ ] Complete the PR description above.
- [ ] Ensure the PR title matches the feature branch name.
- [ ] Check the following:
- [ ]  Instructions provided on how to run
- [ ]  Developer's name is replaced by ${user} where necessary throughout the code
- [ ]  Check that the ecf file has all the proper definitions of variables
- [ ]  Check that the jobs file has all the proper settings of COMIN and COMOUT and other input variables
- [ ]  Check to see that the output directory structure is followed
- [ ]  Be sure that you are not using MET utilities outside the METplus wrapper structure

- [ ] After submitting the PR, select **Development** issue with the original issue number.
- [ ] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
- [ ] Close the linked issue.

### CODE REVIEW

**Comment by ShelleyMelchior-NOAA** [comment]
@ShannonShields-NOAA  I haven't made a run yet for `jevs_subseasonal_prep.ecf` b/c I notice a potential problem. Your script is sourcing `versions/run.ver`, which uses the latest python ve and met-11.0.2. However, your section of module loads is not the correct sequence of loads for the python ve and met-11.0.2.

**Comment by ShannonShields-NOAA** [comment]
Ah yes, sorry, thank you for catching that @ShelleyMelchior-NOAA . I just committed the updated ecf script changes so everything should now be set to go.

**Comment by ShelleyMelchior-NOAA** [comment]
I have kicked off `jevs_subseasonal_prep.ecf`. As of this moment it is still running. Please reivew the output.
`COMOUT=/lfs/h2/emc/vpppg/noscrub/shelley.melchior/EVS_Data/evs/v1.0/prep/subseasonal/atmos`
log file: `/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_subseasonal_prep_00.o57219765`

**Comment by ShannonShields-NOAA** [comment]
It is running the entire prep job (including the models) so it will take about an hour to run. I'll be sure to confirm the output once it is done.

**Comment by ShannonShields-NOAA** [comment]
I can confirm that the prep job ran successfully and that the OSI-SAF file is now reflecting the corrected/updated variables.
I approve this PR for merging.

**Review by ShelleyMelchior-NOAA — APPROVED** [review]
Additional changes were made to ecf scripts to load correct modules for the python ve and MET-11.0.2.

Changes are all correct. Testing results approved by developer.

File: ecf/subseasonal/stats/jevs_cfs_subseasonal_grid2grid_stats.ecf [hunk 1]
```diff
@@ -78,13 +83,13 @@ export cfs_ver=${cfs_ver}
...
 #export COMIN=$COMROOT/$NET/$evs_ver
-export COMIN=/lfs/h2/emc/vpppg/noscrub/$USER/$NET/$evs_ver/prep/$COMPONENT/$RUN
-export COMINobs=/lfs/h2/emc/vpppg/noscrub/$USER/$NET/$evs_ver/prep/$COMPONENT/$RUN
+export COMIN=/lfs/h2/emc/vpppg/noscrub/shannon.shields/EVS_Data/$NET/$evs_ver/prep/$COMPONENT/$RUN
+export COMINobs=/lfs/h2/emc/vpppg/noscrub/shannon.shields/EVS_Data/$NET/$evs_ver/prep/$COMPONENT/$RUN
 export FIXevs=/lfs/h2/emc/vpppg/noscrub/emc.vpppg/verification/EVS_fix
 export COMINclimo=$FIXevs/climos/atmos
-export COMOUT=/lfs/h2/emc/vpppg/noscrub/$USER/$NET/$evs_ver/$STEP/$COMPONENT
+export COMOUT=/lfs/h2/emc/vpppg/noscrub/$USER/evs_subseasonal_test/dev/com/$NET/$evs_ver/$STEP/$COMPONENT
 #export VDATE=$(date -d "today -26 day" +"%Y%m%d")
```
Comment (PerryShafran-NOAA) [inline ecf/subseasonal/stats/jevs_cfs_subseasonal_grid2grid_stats.ecf:94]: Hi, @ShannonShields-NOAA ,

Before I merge into the parallel, I wanted to draw your attention to some of these changes that were made in the ecf files.  Particularly, lines 83 and 84 in red above were changed to lines 88 and 89 in green.  There are other changes made throughout the ecf scripts that were used for testing.  However, they should be changed back to their original lines for parallel usage.  

Can we go through the ecf files to ensure that the various COMROOT, COMIN, COMOUT, etc, are properly set to the way they were for the parallel before?  We may need to submit a new PR for this.  @ShelleyMelchior-NOAA added for awareness.

Thanks!

Perry

Comment (ShannonShields-NOAA) [reply ecf/subseasonal/stats/jevs_cfs_subseasonal_grid2grid_stats.ecf:94]: @PerryShafran-NOAA Ah yes, I see what you are saying; sorry for missing that! I will see what I can do within GitHub to correct those ecf lines and submit a new PR.

**Pull request merged**